CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
 
 set(IRODS_CLIENT_VERSION "0.1.0")
-set(IRODS_PACKAGE_REVISION "1")
+set(IRODS_PACKAGE_REVISION "0")
```

Comment:
Are we bumping `IRODS_CLIENT_VERSION`? What are the implications of moving `IRODS_PACKAGE_REVISION` backwards without moving the version forward?

Reply:
we haven't released yet.   this is okay - 0 is correct.